### DISCUSSION TITLE
Interactive elements not working with ViewTransitions in astro

### DISCUSSION THREAD

**Discussion by soleuil** [themesberg/flowbite#778] [Q&A]
I haven't found any mention to this issue or how to solve it but I just added ViewTransitions to my project in Astro and that made the interactive elements not to work. The hamburguer menu in small size screens for the navigation, or the dropdowns from the menu don't work after doing the first transition to a new page.

In the astro docs they are pointing how to reload the js scripts after the transition, but I wonder how it should be done with a flowbite component like the navbar.

Thank you so much.

**Comment by babul** [comment]
Answer here: https://github.com/themesberg/flowbite/issues/685#issuecomment-1763484445
